client/src/components/Home.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaw } from "@fortawesome/free-solid-svg-icons";
+
+const Home = (props) => {
+
+    return (
+        <div>
+            <div className="home-text">   
+                <h1> 
+                    <FontAwesomeIcon className="right-margin" icon={faPaw}/>
+                    Parks for Paws
+                    <FontAwesomeIcon className="left-margin" icon={faPaw}/>
+                </h1>
```

Comment:
headers should not have nested elements - move the icons outside the `h1` and use font awesome or CSS if you need to adjust the size